javalin/src/main/java/io/javalin/http/servlet/JavalinServletContext.kt

```diff
@@ -141,7 +142,12 @@ class JavalinServletContext(
         userFutureSupplier = future
     }
 
-    override fun skipRemainingHandlers(): Context = also { tasks.clear() }
+    override fun skipRemainingHandlers(): Context = also {
+        tasks.clear()
+        if (handlerType() == WEBSOCKET_BEFORE_UPGRADE) {
+            throw EndOfLifecycleException()
+        }
+    }
```

Comment:
Well originally I was going to avoid supporting this method altogether but @tipsy convinced me otherwise. My alternative implementation would be a sort of "before upgrade context" which would just throw the exception instead and otherwise delegating to the Javalin Servlet context.

Reply:
Support for this method is just fine, I just think it'd be nice if it could work like that without bringing websocket impl details there. I think this method should work out of the box, if handling in `JavalinJettyServlet` would work on `ctx.tasks`

Reply:
Change was reverted with the ctx.tasks replacement.